packages/src/components/editable/editable.component.ts

```diff
@@ -874,18 +1303,33 @@ export class SlateEditableComponent implements OnInit, OnChanges, OnDestroy, Aft
         const { activeElement } = root;
         if (
             !this.readonly &&
-            hasEditableTarget(editor, event.target) &&
-            !isTargetInsideVoid(editor, activeElement) && // stop fire keydown handle when focus void node
-            !this.isComposing &&
-            !this.isDOMEventHandled(event, this.keydown)
```

Comment:
These conditions cannot be removed at will, they all have their own problems to deal with